src/common/transformations/src/ov_ops/rms.cpp

```diff
@@ -34,6 +34,6 @@ std::shared_ptr<Node> RMS::clone_with_new_inputs(const ov::OutputVector& new_arg
                                  m_epsilon);
```

Comment:
If we decide to keep `m_output_type` parameter for the op, then it should be passed as argument here